src/StaticRuleHandlerResolver.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Validator;
+
+use Yiisoft\Validator\Exception\RuleHandlerNotFoundException;
+use Yiisoft\Validator\Rule\RuleHandlerInterface;
+
+final class StaticRuleHandlerResolver implements RuleHandlerResolverInterface
```

Comment:
We need add definition of `RuleHandlerResolverInterface` to container configuration with all default handlers as parameters.